components/BundleModal.tsx

```diff
@@ -247,9 +247,11 @@ function BundleTransaction(transaction, index: number) {
                 }
                 <b>{ coin }</b>
               </a>
-              { coins[coin].value > 0 ? " " + coins[coin].value : "" }
-              { coins[coin].ethValue > 0 ? " ($"+coins[coin].ethValue +")" : "" }
-              { coins[coin].value ? "" : ` (${coins[coin].event})` }
+              <span style={{ marginLeft: coins[coin].ethValue > 0 ? 17 : 0}}>
+                { coins[coin].value > 0 ? ` ${coins[coin].value}` : "" }
+                { coins[coin].ethValue > 0 ? ` ($${coins[coin].ethValue})` : "" }
+                { coins[coin].event ?  ` (${coins[coin].event})` : "" }
```

Comment:
previous version used `coins[coin].value` to decide whether to display the event name as a fallback for lack of price. When there is a price, it becomes too bloated:
![image](https://user-images.githubusercontent.com/1703831/121566764-34cd3900-c9f4-11eb-95be-7337b41e0a04.png)
vs
![image](https://user-images.githubusercontent.com/1703831/121566744-30088500-c9f4-11eb-986e-08e3bcec7547.png)

Reply:
can observe the overlapping in small screen size

Reply:
Yes I observe it too but this comment is about the event (Transfer, Approve, etc). This particular change is showing the event name no matter what, doing it conditional to the lack of value was the preferred approach for legibility.